client/components/ChatMessage.js

```diff
@@ -4,7 +4,14 @@ import { makeStyles } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme) => ({
   guessName: {
```

Comment:
nit: maybe call this `userName` now that it's got its own element?

Reply:
Done.